packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-output.vue

```diff
@@ -72,10 +72,10 @@
 import { ref, onMounted, watch } from 'vue';
 import {
 	FunmanProcessedData,
-	getQueries,
 	processFunman,
 	renderFumanTrajectories
 } from '@/services/models/funman-service';
+import { getRunResult } from '@/services/models/simulation-service';
```

Comment:
Why are you leveraging the simulation endpoint here?

Reply:
Because it has the same mechanism as TA3 "Simulations", that there is an object that encapsulates the running process, which will also track the location(s) of the final result(s). Rather than duplicating the typing and service layer, it makes sense to just multiplex Simulation by adding a "validation" type on top of (ensemble, calibrate, optimize ... etc).